posts/2019/04/uarray-intro.ipynb

```diff
@@ -0,0 +1,473 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# `uarray`: A Generic Override Framework for Methods\n",
+    "\n",
+    "`uarray` is an override framework for methods in Python. In the scientific Python ecosystem, and in other similar places, there has been one recurring problem: That similar tools to do a job have existed, but don't conform to a single, well-defined API. `uarray` tries to solve this problem in general, but also for the scientific Python ecosystem in particular, by defining APIs independent of their implementations.\n",
+    "\n",
+    "## The Scientific Python Ecosystem: How Fragmentation Emerged\n",
+    "When SciPy was created, and Numeric and NumArray unified into NumPy, it jump-started Python's data science community. The ecosystem grew quickly: Academics started moving from MATLAB to SciPy, and the Scikits that popped up made the transition all the more smooth.\n",
```

Comment:
NumArray --> Numarray

Academics lower-case

Reply:
I'd prefer to avoid mentioning Matlab, there's a thing with using the name (in the numpy/scipy docs that has to have a trademark symbol with it everywhere).